routes/membership.js

```diff
@@ -0,0 +1,123 @@
+
+stripe = require("stripe")("sk_test_BQokikJOvBiI2HlWgH4olfQ2");
+module.exports = function(app) {
+
+    app.get('/membership/create', function(req, res, next) {
+        console.log("test");
+        res.render('payment/paywall.jsx');
+    });
+
+
+
+//single token that represents the info the customer has
+{
+  id: "tok_u5dg20Gra", // String of token identifier,
+  card: {...}, // Dictionary of the card used to create the token
+  created: 1447356796, // Integer of date token was created
+  currency: "usd", // String currency that the token was created in
+  livemode: true, // Boolean of whether this token was created with a live or test API key
+  object: "token", // String identifier of the type of object, always "token"
+  used: false // Boolean of whether this token has been used
+}
+
+//sending form to the server
+function stripeResponseHandler(status, response) {
+  var $form = $('#payment-form');
+
+  if (response.error) {
+    // Show the errors on the form
+    $form.find('.payment-errors').text(response.error.message);
+    $form.find('button').prop('disabled', false);
+  } else {
+    // response contains id and card, which contains additional card details
+    var token = response.id;
+    // Insert the token into the form so it gets submitted to the server
+    $form.append($('<input type="hidden" name="stripeToken" />').val(token));
+    // and submit
+    $form.get(0).submit();
+  }
+};
+
+
+
+
+
+/*stripe = require("stripe")("sk_test_BQokikJOvBiI2HlWgH4olfQ2");
```

Comment:
if we're commenting this out, we should go ahead and delete it instead of committing it.